content/applications/finance/fiscal_localizations/thailand.rst

```diff
@@ -109,3 +109,57 @@ in the app, open the **contact form** of the company and under the :guilabel:`Sa
 
 .. tip::
    This information is used in the **tax invoice** PDF report and PND **tax report** export.
+
+PromptPay QR code on invoices
+=============================
+
+The **PromptPay QR code** is a QR code that can be added to invoices to allow customers to pay
+their bills using the PromptPay-supported bank mobile application. The QR code is generated based on
+the **invoice amount** and one of the following **merchant information**:
+
+- Ewallet ID
+- Merchant Tax ID
+- Mobile Number
+
+Activate QR codes
+-----------------
+
+Go to :menuselection:`Accounting --> Configuration --> Settings`. Under the :guilabel:`Customer
+Payments` section, activate the :guilabel:`QR Codes` feature.
+
+PromptPay QR bank account configuration
+---------------------------------------
+
+Go to :menuselection:`Contacts --> Configuration --> Bank Accounts` and select the bank account for
+which you want to activate PromptPay QR. Set the :guilabel:`Proxy Type` and fill in the
+:guilabel:`Proxy Value` field depending on the chosen type.
+
+.. important::
+   - The :guilabel:`Include Reference` checkbox doesn't work for PromptPay QR codes.
+
+.. image:: thailand/thailand-promptpay-bank-setting.png
+   :alt: PromptPay bank account configuration
+
+.. seealso::
+   :doc:`../accounting/bank`
+
+Bank journal configuration
+--------------------------
+
+Go to :menuselection:`Accounting --> Configuration --> Journals`, open the bank journal, then fill
+out the :guilabel:`Account Number` and :guilabel:`Bank` under the :guilabel:`Journal Entries` tab.
+
+.. image:: thailand/thailand-bank-account-journal-setting.png
```

Comment:
For example here, this name could apply to other topics than the QR code section.
```suggestion
.. image:: thailand/qr-bank-journal.png
```